lametro/models.py

```diff
@@ -153,6 +153,56 @@ def board_report(self):
 
         return br
 
+    @property
+    def actions_and_agendas(self):
+        '''
+        Return list of dictionaries, each representing an action or an agenda on
+        which the given bill appears, in the format:
+
+        {
+            'date': <datetime>,
+            'description': <str>,
+            'event': <LAMetroEvent>,
+            'organization': <Organization>
+        }
+        '''
+        actions = self.actions.all()
+        events = LAMetroEvent.objects.filter(agenda__related_entities__bill=self)
+
+        data = []
+
+        for action in actions:
+            event = LAMetroEvent.objects.filter(\
```

Comment:
`filter()` returns a queryset – we want to attach the resulting event instance, not the queryset, to the action dict. I would expect this to only ever return one result. How about using `get()` here?

```suggestion
            event = LAMetroEvent.objects.get(\
```